app/controllers/users/passwords_controller.rb

```diff
@@ -66,11 +78,9 @@ def confirm_valid_token
     end
 
     def handle_successful_password_reset_for(resource)
-      resource.unlock_access! if unlockable?(resource)
```

Comment:
Yes, we are dropping it. This is one of many examples of Devise conditionals that will always return false in our app.